### PR TITLE
(IT) add integration tests for review controller + Fix test configuration

### PR DESCRIPTION
**Pull request by RaduADumitru**
- Added integration tests for review controller: sample smoke test and testing Spring Security
- Fixed configuration for integration tests; added test Postgres container config
- Added debug logging to `testIT` Gradle task

### CODE REVIEW

File: src/test/java/ro/unibuc/triplea/application/games/steam/web/SteamGameControllerTestIT.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+public class SteamGameControllerTestIT {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private SteamGameService steamGameService;
```
Comment (nicugnm) [inline src/test/java/ro/unibuc/triplea/application/games/steam/web/SteamGameControllerTestIT.java:31]: do not use @Autowired in tests. class that wants to be tested should be instantiated and use mock parameters.

File: src/test/java/ro/unibuc/triplea/application/games/steam/web/SteamGameControllerTestIT.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+    @Test
+    public void testGetGameByIdentifier() throws Exception {
+        // Given
```
Comment (nicugnm) [inline src/test/java/ro/unibuc/triplea/application/games/steam/web/SteamGameControllerTestIT.java:35]: delete comments as they are not useful.

File: src/test/java/ro/unibuc/triplea/application/games/steam/web/SteamGameControllerTestIT.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                .andReturn();
+
+        // Then
+        String responseBody = result.getResponse().getContentAsString();
+        assertNotNull(responseBody);
+
+        SteamGameResponse actualGame = objectMapper.readValue(responseBody,
+                SteamGameResponse.class);
+        assertEquals(expectedGame, actualGame);
+    }
```
Comment (RaduADumitru) [inline src/test/java/ro/unibuc/triplea/application/games/steam/web/SteamGameControllerTestIT.java:57]: This test will be fixed in a separate PR, it was leftover from another branch

File: src/test/resources/application.yml [hunk 4]
```diff
@@ -0,0 +1,34 @@
+spring:
+  datasource:
+    url: jdbc:tc:postgresql:11.1:///integration-tests-db
```
Comment (nicugnm) [inline src/test/resources/application.yml:3]: why do we use this url?

**Review by nicugnm — CHANGES_REQUESTED** [review]

File: src/test/resources/application.yml (hunk 4 above)
Comment (RaduADumitru) [reply src/test/resources/application.yml:3]: This needs to be used to connect to the Postgres test container

File: src/test/java/ro/unibuc/triplea/application/games/steam/web/SteamGameControllerTestIT.java (hunk 3 above)
Comment (nicugnm) [reply src/test/java/ro/unibuc/triplea/application/games/steam/web/SteamGameControllerTestIT.java:57]: why not delete it and added in proper branch? we don't polute master with failed tests

File: src/test/java/ro/unibuc/triplea/application/games/steam/web/SteamGameControllerTestIT.java (hunk 1 above)
Comment (RaduADumitru) [reply src/test/java/ro/unibuc/triplea/application/games/steam/web/SteamGameControllerTestIT.java:31]: As I mentioned above this class is a leftover from another branch, the test here will be done by @alexbanilean . I can remove it from this PR to avoid confusion, and it will be added back when it is complete

**Review by nicugnm — APPROVED** [review]

**Review by alexbanilean — APPROVED** [review]
